test/pack_test.cpp

```diff
@@ -234,98 +234,98 @@ const ItemPack PackedDiabloItems[] = {
-	{ "Soldier's Sword of vigor",     ItemType::Sword,            1,     57,    31600,        6,       15,    0,          0,        0,       0,         0,            0,           60,       60,      66,        19,      0,       0,       0,       0,      20,      0,      0,      0,        0,      0,          0,          0,         0,           0,     0,         0,         0,         0,         0,          4,         25,       50,        0,        0,   127 },
-	{ "Fine Long Bow of the pit",     ItemType::Bow,              1,    102,     2152,        1,        6,    0,          0,        0,       0,         0,            0,           35,       35,      49,        10,      0,      -2,      -2,      -2,      -2,      0,      0,      0,        0,      0,          0,          0,         0,           0,     0,         0,         0,         0,         0,          4,         28,       25,        0,       30,   145 },
-	{ "Sharp Sword of atrophy",       ItemType::Sword,            1,     60,     1958,        2,       10,    0,          0,        0,       0,         0,            0,           24,       40,      34,         4,      0,       0,       0,      -1,       0,      0,      0,      0,        0,      0,          0,          0,         0,           0,     0,         0,         0,         0,         0,          4,         24,       30,        0,       30,   125 },
-	{ "Emerald Bow of burning",       ItemType::Bow,              1,    120,   107000,        1,       14,    0,          8,        0,       0,         0,            0,           60,       60,       0,         0,      0,       0,       0,       0,       0,     50,     50,     50,        0,      0,          0,          0,         0,           0,     0,         1,        16,         0,         0,         11,         42,       45,        0,       80,   150 },
+	//_iIName,                        _itype,                _iClass, _iCurs, _iIvalue, _iMinDam, _iMaxDam, _iAC,                                    _iFlags, _iMiscId, _iSpell, _iCharges, _iMaxCharges, _iDurability, _iMaxDur, _iPLDam, _iPLToHit, _iPLAC, _iPLStr, _iPLMag, _iPLDex, _iPLVit, _iPLFR, _iPLLR, _iPLMR, _iPLMana, _iPLHP, _iPLDamMod, _iPLGetHit, _iPLLight, _iSplLvlAdd, _iUid, _iFMinDam, _iFMaxDam, _iLMinDam, _iLMaxDam, _iPrePower, _iSufPower, _iMinStr, _iMinMag, _iMinDex, IDidx );
+	{ "Amber Helm of harmony",        ItemType::Helm,              2,     98,    21100,        0,        0,   11,    static_cast<ItemSpecialEffect>(8388608),        0,       0,         0,            0,           60,       60,       0,         0,      0,       0,       0,       0,       0,     18,     18,     18,        0,      0,          0,          0,         0,           0,     0,         0,         0,         0,         0,         11,         59,       50,        0,        0,    53 },

```

Comment:
These values are combinations of flags, should be able to compose them from the named enum values.

Reply:
Yeah, I was a bit lazy on this one, but also was concerned about breaking the table for a few items with a bunch of values. This keeps it a little more compact. If we really want it converted to the actual flag values, I guess I can script it.

Reply:
Looks like only Griswold's Edge and Empyrean Band use more than one flag, giving them named constants would let the table be a list of single flag values otherwise. That'd work fine.

Reply:
done, although there were 4 more in the other table too 🙂